File: app/src/main/java/com/rdev/tryp/model/ride_responce/RideResponse.java
package com.rdev.tryp.model.ride_responce;

import com.google.gson.annotations.SerializedName;

public class RideResponse {

    @SerializedName("data")
    private Data data;

    public void setData(Data data) {
        this.data = data;
    }

    public Data getData() {
        return data;
    }

    @Override
    public String toString() {
        return
                "RideResponse{" +
                        "data = '" + data + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/rdev/tryp/model/ride_responce/RideResponse.kt
package com.rdev.tryp.model.ride_responce

import com.google.gson.annotations.SerializedName


class RideResponse {

    @SerializedName("data")
    var data: Data? = null

    override fun toString(): String {
        return "RideResponse{" +
                "data = '" + data + '\''.toString() +
                "}"
    }

}